### ISSUE TITLE
No animation when transitioning between nested navigators

### ISSUE THREAD

**Issue by LuisRodriguezLD** [elyalvarado/react-navigation-switch-transitioner#10]
Hello.

I managed to make it work using [this fork](https://github.com/codebutler/react-navigation-switch-transitioner/tree/fix-issues).

But I have two pages which are not transitions smoothly, I believe the reason is because those pages have a stacknavigator on their own.

Any workaround? Maybe I am doing something wrong. Here is small example.

```js
const AppNavigator = createSwitchNavigator({
	Loading: withTransition(FadeTransition)(loading), // page
	Welcome: withTransition(FadeTransition)(welcome), // page
	LoggedOut: withTransition(FadeTransition)(loggedout), // nav
	LoggedIn: withTransition(FadeTransition)(loggedin), // nav
});
```

this is how a nav looks 

```js
const Navigator = createBottomTabNavigator({
	Home: home,
        ...
})
```

**Comment by elyalvarado** [comment]
Yes, this has been an issue since the beginning. I fixed some cases in version 0.1.1, and it'll work if you don't wrap the nested navigator in the withTransition HOC. However, there is a caveat: it'll only work when transitioning from/to a wrapped screen, in your case it'll work if you transition from loading to loggedIn, or from welcome to loggedIn. It won't work if you transition between loggedOut and LoggedIn.

**Comment by LuisRodriguezLD** [comment]
You are correct. It works precisely as you described.
Do you know a workaround this issue?

**Comment by elyalvarado** [comment]
not yet, the only workaround is what is currently implemented

**Comment by JeanMonteiro** [comment]
same here!

**Comment by mahdi-nvbi** [comment]
same.
